src/ics/converter/types/calendar.py

```diff
@@ -19,7 +19,8 @@ class CalendarMeta(ComponentMeta):
 
```

Comment:
Could you update the docstring above explaining [this](https://github.com/ics-py/ics-py/pull/312/files/bafba59bb873105f76798105343878d43659e2a8#r805589474) ?

Reply:
Done